.github/workflows/alpine_image.yaml

```diff
@@ -0,0 +1,63 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Build bevel-build docker image triggered by :
+#  - push on develop branch for latest tags 
+#  - create tag vx.y.z for release tags
+#  - Can be triggered manually
+
+name: Build alpine1.1 Docker image and push
+
+on:
+  workflow_dispatch:
+  create:
+    tags:
+      - v*
+  push:
+    branches:
+      - develop
+    paths:
+      - '.github/workflows/alpine_image.yml'
+      - 'platforms/shared/images/alpine-utils.Dockerfile'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PATH: ghcr.io/${{ github.repository_owner }}
+
+jobs:
+  # on push to develop branch, create docker tags latest and jdk8-latest
+  alpine-build-latest:
+    if: ${{ startsWith(github.ref, 'refs/heads/develop') }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Login to the container registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+    - name: Lowercase IMAGE_PATH
+      id: image_path_format
+      uses: ASzc/change-string-case-action@v2
+      with:
+        string: ${{ env.IMAGE_PATH }}
+
+    - name: Build and push release tag latest
+      id: alpine_build_latest
+      uses: docker/build-push-action@v2
+      with:
+        context: .
+        file: ./platforms/shared/images/alpine-utils.Dockerfile
+        push: true
+        tags: ${{ steps.image_path_format.outputs.lowercase }}/alpine-utils:1.1
```

Comment:
the tag should be based on the release tag for releases, latest for develop branch push